Podfile

```diff
@@ -211,7 +211,7 @@ abstract_target 'Apps' do
 
     pod 'Gridicons', '~> 1.1.0'
 
-    pod 'WordPressAuthenticator', '~> 1.38.0'
+    pod 'WordPressAuthenticator', '~> 1.40.0-beta'
```

Comment:
This `Podfile` change is missing its counterpart `Podfile.lock` update.

Every time we change the `Podfile`, we should also run `bundle exec pod install`. 

This is also what [the failure in CI is about](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-iOS/22840/workflows/51587738-2c04-4ef6-aa81-2ca6a52b0fd2/jobs/52609).

Reply:
> Every time we change the `Podfile`, we should also run `bundle exec pod install`.

Ah, good to know! So since I changed the `Podfile`, when I run `bundle exec pod install` it checks to see what versions of things are in the `Podfile`, and updates `podfile.lock` according to that. Is that right?

Reply:
Yes.

In the `Podfile`, you can define dependency resolution rules, such as `> 1.2`, which will install any version greater than 1.2.

In the `Podfile.lock`, CocoaPods writes the _resolved_ version, the one it actually ended up installing.

When we run `pod install`, CocoaPods does something along the lines of:

1. Checks to see if the checksum is the same between `Podfile` and `Podfile.lock`
2. If it is the same (no changes have been made to the `Podfile`), downloads and installs all the versions as specified by `Podfile.lock`
3. If it is not the same (something changed in the `Podfile`), it resolves the dependencies based on what's defined in the `Podfile`, writes the result in the `Podfile.lock`, then downloads and installs them

Dependency resolution is useful in cases such the following. A project uses three libraries: A, B, and C. Library A uses library B at version 1.1; library C also uses library B, but at version 1.2. Depending on the dependency resolution rules configured in the `Podfile` and in each library, the project might end up installing library B version 1.2 or fail the setup and tell the user they need to update library A to use library B at version 1.2, too, to ensure compatibility.

_Dependency management and resolution is a complex topic._

Reply:
> Dependency management and resolution is a complex topic. 

💯 So it seems. Thanks!